.lintstagedrc

```diff
@@ -1,5 +1,6 @@
 {
   "*.js": [
+    "git add .",
```

Comment:
Because it's one less manual step that speeds up development and makes people work in smaller chunks. https://github.com/okonet/lint-staged#examples